toolkit/scripts/pkggen.mk

```diff
@@ -26,7 +26,8 @@ pkggen_rpms     = $(call shell_real_build_only, find $(RPMS_DIR)/*  2>/dev/null
 cache_working_dir      = $(PKGBUILD_DIR)/tdnf_cache_worker
 parse_working_dir      = $(BUILD_DIR)/spec_parsing
 rpmbuilding_logs_dir   = $(LOGS_DIR)/pkggen/rpmbuilding
-rpm_cache_files        = $(call shell_real_build_only, find $(CACHED_RPMS_DIR))
+rpm_cache_cache_dir    = $(CACHED_RPMS_DIR)/cache
```

Comment:
The `cache_cache` part of the name is a bit confusing. I think the main problem lies with `CACHED_RPMS_DIR`, since it's not the directory where the RPMs end up. It's more of a top directory for all things cache-related. What do you think about re-naming `rpm_cache_cache_dir` to `remote_rpms_cache_dir`, `rpm_cache_files` to `cached_remote_rpms`, and `CACHED_RPMS_DIR` to `CACHING_TOP_DIR`? Or something along these lines with a focus on clarifying what each variable contains.

Reply:
Agreed, but I don't think I want to change `CACHED_RPMS_DIR` since its a public facing variable... I'll swap the others though.